app/routes/assets.js

```diff
@@ -1,55 +1,59 @@
 const config = require('../config')
 
-module.exports = [{
-  method: 'GET',
-  path: '/static/assets/{path*}',
-  options: {
-    auth: false,
-    handler: {
-      directory: {
-        path: [
-          'node_modules/govuk-frontend/dist/govuk/assets'
-        ]
+module.exports = [
```

Comment:
It looks like the only changes to this file is formatting, no actual code change?

Reply:
correct